src/components/notifications/AppNotifications/AppNotificationsHeader/index.tsx

```diff
@@ -83,7 +82,66 @@ const AppNotificationsHeader: React.FC<IAppNotificationsHeaderProps> = ({ logo,
 
   return (
     <div className="AppNotificationsHeader">
-      <div className="AppNotificationsHeader__content">{HeaderContent}</div>
+      <div className="AppNotificationsHeader__content">
+        {dappOrigin ? (
+          <div className="AppNotificationsHeader__plain">
+            <h2>Notifications</h2>
+            <AppNotificationDropdown
+              dropdownPlacement="bottomLeft"
+              notificationId={id}
+              h="2em"
+              w="2em"
+              closeDropdown={noop}
+            />
+          </div>
+        ) : (
+          <>
+            <div className="AppNotificationsHeader__app">
+              <BackButton backTo="/notifications" />
+              <img
+                className="AppNotificationsHeader__app__logo"
+                src={logo}
+                alt={`${name}logo`}
+                loading="lazy"
+              />
+              <h2 className="AppNotificationsHeader__app__name">{name}</h2>
+            </div>
+            <div className="AppNotificationsHeader__wrapper">
+              {/*
+             <div className="AppNotificationsHeader__selector">
+              <Select
+                name="explorer-selector"
+                id="explorer-selector"
+                onChange={console.log}
+                options={[{ label: 'All', value: 'all' }]}
+              />
+            </div>
```

Comment:
Let's remove dead code, we can look back at this later on. Let's keep a clean codebase